README.md

```diff
@@ -57,5 +84,6 @@ In order to be able to launch the bot server locally you have to compile sources
 
 + `IS_LONGPOLL=true` for using long polling instead of web hooking (common usage case is local launch and debugging)
 
-[![Pull Party Bot](info/logo-white-no_boarders.png)](https://t.me/PullPartyBot)
+
+<img src="info/logo-white-no_boarders.png" align="center">
```

Comment:
It is still not centered 🤪 Needs fix

Reply:
Probably, u can wrap this in a paragraph 🤪